streams/src/test/java/org/apache/kafka/streams/processor/internals/testutil/LogCaptureAppender.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.kafka.streams.processor.internals.testutil;
```

Comment:
I can move them there. The reason I put them here was to restrict the scope in which log4j was an allowed import (I had to add an exception). Co-locating this class in any other package will allow other code to accidently depend on log4j when it should depend on slf4j instead.

I was also uncertain about whether it would be a good idea to expose this class for general use in Kafka tests... WDYT?

If you're not concerned about supporting this class for the whole project, maybe `o.a.k.test.log4jappender` package would be the best of both worlds?

Reply:
Ah I see, overlooked `log4j` dependency; let's keep it as is then.